### ISSUE TITLE
The dropdown options can't be selected using the arrow keys or the Tab key. It doesn't request focus, so it's not accessible via keyboard navigation.

### ISSUE THREAD

**Issue by aaksa** [FlutterFlow/flutterflow-issues#5889]
### Can we access your project?

- [x] I give permission for members of the FlutterFlow team to access and test my project for the sole purpose of investigating this issue.

### Current Behavior

When clicking the dropdown, focus doesn't move directly to the options (they can't be selected using the Tab key or arrow keys). It's important to fix this to comply with the Americans with Disabilities Act (ADA).

### Expected Behavior

After clicking the dropdown, or when it receives focus, we should be able to select an option using the arrow keys or the Tab key.

### Steps to Reproduce

- Create Dropdown with it options
- click or change the focus to dropdown
- select using arrow keys or tab button

### Reproducible from Blank

- [x] The steps to reproduce above start from a blank project.

### Bug Report Code (Required)

IT4oh8/l5clgocdG0L6Jce9VpyQUOUw+RK8n0spBZywgfZzzBLN/fPfQPlpoQuLmTEFIM06Mp14Gp9b9hYSSL/AEKT6ZQKp/ypNiSg3zUjiXbJ+TF7WZP3wkP+RTfR2a156JgRV7Ls9pLV44wUy2AtyAc3TqJpDXIno8Sq/LZO4=

### Visual documentation

this serves as visual documentation : https://drive.google.com/file/d/1J8ggENpwObfS5WC7hDT4mO8aHz6kNMRl/view?usp=sharing

### Environment

```markdown
- FlutterFlow version: v5.5.3
- Platform: Web
- Browser name and version:Chrome Version 136.0.7103.114 (Official Build) (arm64)
- Operating system and version affected: MacOs 15.5 (24F74)
```

**Comment by Alezanello** [comment]
Hi there!

Thanks for reporting this. It looks like this is a Flutter-related issue rather than something specific to FlutterFlow. You can find more details and track progress on the Flutter GitHub issue here: [Flutter Issue #58494](https://github.com/flutter/flutter/issues/58494).

**Comment by Alezanello** [comment]
Closing this issue since we haven't heard back from you. If you are still facing the problem please submit a new issue. Thank you!
